docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,268 @@
+# DALi Widget Application
+
+
+You can create a widget application with DALi to display small version of application on the home screen.
+
+To create a DALi widget application, you must:
+
+- Use the [Dali::WidgetApplication class](#widget-application-api).
+
+	> **Note**
+	> - The implementation of DALi widget application is similar to [DALi basic UI application](dali-basic-app.md), because the [Dali::WidgetApplication](../../api/mobile/latest/classDali_1_1WidgetApplication.html) class inherits from the [Dali::Application](../../api/mobile/latest/classDali_1_1Application.html) class.
+  > - Ensure you are familiar with the basic UI application details before tackling a widget application.
+
+- Ensure you start the event loop.
+
+- To implement your own widget class inherit the `Dali::Widget` class.
+
+![DALi widget application](./media/dali_widget_application.png)
+
+**Figure: DALi widget application**
+
+For more information on basics of creating a DALi widget application, see [The steps to create a DALi widget application](#create).
+
+<a name="widget-application-api"></a>
+## Widget Application Class
+
+If you want to use DALi for your widget application, you must create the application with the `Dali::WidgetApplication` class. This class provides the means for initializing the resources required by DALi.
+
+The table list the main functions to manage widget instance:
+
+**Table: Main functions**
+
+| Function                             | Description                              |
+|--------------------------------------|------------------------------------------|
+| `mainloop()`                         | The mainloop() function is used to start the event loop. If you do not call the function and start the event loop, DALi cannot call any callback functions for application events.  |
+| `RegisterWidgetCreatingFunction()`   | This function is used to register create function to create widget instance. When widget viewer application requests, then the widget application creates widget instance using the function |
```

Comment:
Please change to:
This function is used to register create function to create a widget instance. When the widget viewer application requests, then a widget application creates a widget instance using the function.

Question:
1) What does the widget viewer application requests?
2) Please can you clarify whether the widget application creates a widget instance using `RegisterWidgetCreatingFunction()`   ?

Reply:
1. Widget viewer app requests making widget instance to widget application. If widget application get request, it will make widget instance by using creation function which is registered by widget application developer.

2. RegisterWidgetCreatingFunction is function for registering creation function to widget application. It is kind of factory method pattern(https://en.wikipedia.org/wiki/Factory_method_pattern).

Reply:
@neostom432, thank you for the detail explanation. 

Please change to:
 This function is used to register create function to create a widget instance. When the widget viewer application requests, then a widget application creates a widget instance using the function.